infra/base-images/base-runner/bad_build_check

```diff
@@ -306,22 +306,21 @@ function check_mixed_sanitizers {
     return 0
   fi
 
-  if [[ $ARCHITECTURE == 'i386' ]]
+  if [[ $ARCHITECTURE == "x86_64" ]]
+  then
+    CALL_INSN="callq\s+[0-9a-f]+\s+<"
+    OBJDUMP="objdump"
+  elif [[ $ARCHITECTURE == "i386" ]]
   then
     CALL_INSN="call\s+[0-9a-f]+\s+<"
+    OBJDUMP="objdump"
+  elif [[ $ARCHITECTURE == "aarch64" ]]
+  then
+    CALL_INSN="bl\s+[0-9a-f]+\s+<"
+    OBJDUMP="aarch64-linux-gnu-objdump"
```

Comment:
This binary exists already in our image?

Reply:
Oh this is was copied by accident I can get rid of this ENV var.